src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/v2/common.kt

```diff
@@ -299,6 +299,8 @@ fun patchAuthRequestByState(
                   authorizationCode =
                     stateResponseDto.operation!!.additionalData!!["authorizationCode"] as String?
                   rrn = stateResponseDto.operation!!.additionalData!!["rrn"] as String?
+                  validationServiceId =
```

Comment:
Here we are missing also `errorCode` field propagation.

What about add it here? Or maybe we can open a separate PR/task